test/e2e/storage/csi_volumes.go

```diff
@@ -84,7 +85,18 @@ var _ = Describe("E2E", func() {
 		testsuites.InitVolumeIOTestSuite,
 		testsuites.InitVolumeModeTestSuite,
 		testsuites.InitVolumesTestSuite,
-		testsuites.InitEphemeralTestSuite,
+	}
+	if k8sVersion := os.Getenv("TEST_KUBERNETES_VERSION"); k8sVersion != "" {
+		var major, minor int
+		if _, err := fmt.Sscanf(k8sVersion, "%d.%d", &major, &minor); err != nil {
+			framework.Logf("Failed to parse Kubernetes version '%s': %s", k8sVersion, err.Error())
+		}
+		// Run ephemeral test only on supported Kubernetes versions, i.e., 1.14+
+		if major >= 1 && minor > 14 {
```

Comment:
Please invert the logic: always add the test unless `TEST_KUBERNETES_VERSION` was set to something <= 1.14. Otherwise they might get skipped accidentally.

Reply:
Rewrite the logic to support your suggestion.